packages/storybook-cmf/package.json

```diff
@@ -40,7 +40,7 @@
     "eslint-plugin-react": "^6.3.0",
     "immutable": "3.8.1",
     "invariant": "2.2.2",
-    "jest": "20.0.3",
+    "jest": "^23.0.0",
```

Comment:
Why do we have sometimes `"^23.0.0"` and othee times `"^23.6.0"` ?
We should align them